docs/getting-started.md

```diff
@@ -0,0 +1,161 @@
+
+Where:
+```provinces_of_brazil = client.get_descendant_regions(1029, 4)
+```
+will recursively lookup all descendants of region 1029 (Brazil) that are of level 4 (provinces) i.e. all the provinces of Brazil. 
+
+And: 
+```districts_of_brazil = client.get_descendant_regions(1029, 5)
+```
+will recursively lookup all descendants of region 1029 (Brazil) that are of level 5 (districts), i.e. all the districts in all the provinces of Brazil.
+
+For more information on region levels, please refer to the [Special properties for regions](../docs/gro-data-key-concepts#special-properties-for-regions) section of [Gro Data: Key Concepts]((../docs/gro-data-key-concepts)
+
+### Lookup belongs
+If you want to find "what entities contain the given entity?" there is a helper function, `client.lookup_belongs(type, child_id)` to does just that. For example:
+
+`client.lookup_belongs('regions', 1215)`
+will return `[{id: 15, name: 'North America', contains: [1215, 1037, ...], level: 2}, ...]`
+
+## Retrieving Data
+Once you've identified the data you want, you'll want to start retrieve it and start putting it to use. The follow methods should help you get the data in the format that you want.
+
+### Get data points
+`get_data_points(**selection)` is the primary method for retrieving data. The [code snippets](#code-snippets) feature covered earlier provides you with a fully completed `get_data_points` query, such as:
+```# Wheat - Area Harvested (area) - India (USDA PS&D)
+client.get_data_points(**{'metric_id': 570001, 'item_id': 95,'region_id': 1094, 'source_id': 14, 'frequency_id': 9})
+```
+The above query has completed fields for `metric_id`, `item_id`, `region_id`, `source_id`, and `frequency_id`. However, `get_data_points` can also accept fields to further narrow your data series of interest: `partner_region_id` (used only in series that represent a flow between two places), `start_date`, `end_date`, and [`show_revisions`](#show-revisions).
+
+At a minimum, get_data_points must have a `metric_id` and a `source_id` specified. Omitting some of the default fields can help you retrieve greater amounts of data in a single query. For example, removing the `item_id` from the above query:
+```client.get_data_points(**{'metric_id': 570001, 'region_id': 1094, 'source_id': 14, 'frequency_id': 9})```
+Will return the Area Harvested series for all items in India as available from USDA PS&D.
+
+Note that limiting the specificity of your selection can greatly increase the time it takes for a response to be returned.
+
+### Data frame
+Data frames are popular format for viewing data responses, and our `gro_client` library offers you the ability view your data series in a data frame. If you've imported the library into your file, like:
+```import api.client.gro_client
+from api.client.gro_client import get_df

```

Comment:
omit the imports here since the rest of this doc already assumes you have client so its better t just refer to client.get_df() here

Reply:
This is the only method mentioned in this file that isn't in the main library, though. I could see some users just doing the basic set up and then having this method not work out for them if they didn't realize it was in gro_client.

Reply:
This is the deprecated version of this function though. Should not be encouraging use of it

See the warning here: https://github.com/gro-intelligence/api-client/blob/development/api/client/gro_client.py#L297

Use `client.get_df()` instead.

We should add a deprecation warning print to the console too, like I did for the api.client renaming PR here: https://github.com/gro-intelligence/api-client/pull/63

Reply:
Great catch!